package.json

```diff
@@ -52,9 +52,6 @@
     "@babel/core": "^7.15.8",
     "@itwin/eslint-plugin": "3.3.5",
     "@types/node": "10.14.1",
```

Comment:
What's the need for this and babel/core?

Reply:
Apparently nothing; removed.